ts/@live-compositor/browser-render/examples/counter/src/App.tsx

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useRef, useState } from 'react';
+import './App.css';
+import { Renderer, loadWasmModule } from '@live-compositor/browser-render';
+
+function App() {
+  const canvasRef = useRef<HTMLCanvasElement | null>(null);
+  const [renderer, setRenderer] = useState<Renderer | null>(null);
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
```

Comment:
refactor parts of the code to separate hook or just outside the component (it's fine to keep it in the same file for now)